### PR TITLE
Invite from where popup, FloatingMenu refactor

### PR DESCRIPTION
**Pull request by jakob223**
- "invite from where" popup on mobile
 - simplify the `header` prop on `FloatingMenu`

cc @keybase/y2ksquad

### CODE REVIEW

**Comment by jakob223** [comment]
I recommend hiding whitespace changes https://github.com/keybase/client/pull/23152/files?diff=split&w=1

File: shared/chat/conversation/messages/message-popup/exploding/index.tsx [hunk 1]
```diff
@@ -164,14 +164,7 @@ class ExplodingPopupHeader extends React.Component<Props, State> {
 }
 
 const ExplodingPopupMenu = (props: Props) => {
-  const header = {
-    style: {
```
Comment (buoyad) [inline shared/chat/conversation/messages/message-popup/exploding/index.tsx]: was this needed?

File: shared/people/invite-friends/invite-how.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import * as React from 'react'
+import * as Kb from '../../common-adapters'
+import * as Container from '../../util/container'
+
+type Props = Pick<React.ComponentProps<typeof Kb.FloatingMenu>, 'attachTo' | 'visible' | 'onHidden'>
```
Comment (buoyad) [inline shared/people/invite-friends/invite-how.tsx]: don't think it needs `attachTo`

File: shared/router-v2/tab-bar.desktop.tsx [hunk 3]
```diff
@@ -86,33 +86,29 @@ const Header = () => {
   const onSettings = () => dispatch(RouteTreeGen.createSwitchTab({tab: Tabs.settingsTab}))
   const onSignOut = () => dispatch(RouteTreeGen.createNavigateAppend({path: [SettingsConstants.logOutTab]}))
 
-  const menuHeader = () => ({
-    onClick: onProfileClick,
```
Comment (buoyad) [inline shared/router-v2/tab-bar.desktop.tsx]: Not needed?

File: shared/util/feature-flags.native.tsx [hunk 4]
```diff
@@ -9,7 +9,7 @@ const ff: FeatureFlags = {
   admin: __DEV__,
   connectThrashCheck: false,
   foldersInProfileTab: false,
-  inviteFriends: false,
+  inviteFriends: true,
```
Comment (buoyad) [inline shared/util/feature-flags.native.tsx]: :fire:

File: shared/chat/conversation/messages/message-popup/exploding/index.tsx (hunk 1 above)
Comment (jakob223) [reply shared/chat/conversation/messages/message-popup/exploding/index.tsx]: It was ignored by the FloatingMenu, so I hope not :P

File: shared/router-v2/tab-bar.desktop.tsx (hunk 3 above)
Comment (jakob223) [reply shared/router-v2/tab-bar.desktop.tsx]: Same as above, FloatingMenu ignored it

File: shared/util/feature-flags.native.tsx (hunk 4 above)
Comment (jakob223) [reply shared/util/feature-flags.native.tsx]: 🤦

**Pull request merged**